abstract-media-service/src/main/java/org/arpha/service/BlobService.java

```diff
@@ -0,0 +1,8 @@
+package org.arpha.service;
+
+import org.arpha.dto.media.enums.AccessType;
+
+public interface BlobService {
+
+    String generateLink(String fileName, AccessType accessType);
+}
```

Comment:
Do we need this abstraction over BlobServiceImpl? I am not sure that we actually want to expose BlobService itself.